easybuild/easyconfigs/s/spglib/spglib-1.9.4.2-intel-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = "PythonPackage"
+
+name = 'spglib'
```

Comment:
@boegel sorry again for the delay - related but more generic, shouldn't the domain in the `PYPI_SOURCE` template be changed to `pypi.io` or `files.pythonhosted.org` instead of `pypi.python.org`? e.g. fedora-python/pyp2rpm#50